daemon/github.go

```diff
@@ -83,7 +83,7 @@ func processPushEvent(event *github.PushEvent) {
 			return
 		}
 		defer cli.Close()
-		err = deploy(localRepo, branch, cli, os.Stdout)
+		err = deploy(localRepo, branch, *repo.Name, cli, os.Stdout)
```

Comment:
@bobheadxi 

What do you think about using `repo.Name` here? In the other call to `deploy()`, we use the project folder name e.g. `inertia-deploy-test`, but we wouldn't expect them to differ?

Reply:
Ah, forgot about this one.

What I might do is save `projectName` as a variable and always use that - I think that should be safe, since automated deployment can not happen until we receive an `up` request, which will include the `project` in its payload.